tools/windows/install-help/cal/LogonCli.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <Windows.h>
+#include "NonCopyable.h"
+
+class LogonCli : private NonCopyable<LogonCli>
+{
+  private:
+    typedef NTSTATUS (*SigNetIsServiceAccount)(_In_opt_ LPWSTR, _In_ LPWSTR, _Out_ BOOL *);
+    HMODULE _logonCliDll;
+    SigNetIsServiceAccount _fnNetIsServiceAccount;
+
+  public:
+    LogonCli();
+    LogonCli(LogonCli &&other) noexcept;
+    LogonCli &operator=(LogonCli &&) noexcept;
+    ~LogonCli();
+
+    NTSTATUS NetIsServiceAccount(_In_opt_ LPWSTR ServerName, _In_ LPWSTR AccountName, _Out_ BOOL *IsService);
+};
```

Comment:
Surprised that you are doing rather sophisticated protection and copying but use the class as a raw pointer instead of shared_ptr or probably better here unique_ptr

Reply:
do not bother just a note